sass/variables/_sizing.scss

```diff
@@ -73,6 +73,12 @@ $carousel-dot-spacing: 10px;
 $cta-icon-size: 2rem;
 $accordion-icon-size: 30px;
 $resource-continue-block-size: 40px;
+$visual-divider-endpoint-size: 14px;
```

Comment:
per story I think this should be 13

![image](https://cloud.githubusercontent.com/assets/2359538/26640360/2524e320-45ed-11e7-8695-dff7fa32380c.png)